docs/source/contrib/metrics.rst

```diff
@@ -1,56 +1,15 @@
 ignite.contrib.metrics
-======================
+=======================
 
-Contrib module metrics
-----------------------
+Contrib module metrics [deprecated]
+-----------------------------------
 
-.. currentmodule:: ignite.contrib.metrics
+.. deprecated:: 0.4.14
```

Comment:
I was thinking to increment minor version with a lot of contrib moves -> `0.5.1`
`0.5.0` was spoiled on pypi when we accidentally released a nightly version as 0.5.0